docs/contributing/documentation_guidelines.md

```diff
@@ -82,6 +82,17 @@ Make sure to include the correct language code for syntax highlighting, and to f
     ]
     ```
 
+<details>
+    <summary>Rendered output</summary>
+    ```python<br>
+    INSTALLED_APPS = [<br>
+        ...<br>
+        "wagtail",<br>
+        ...<br>
+    ]
+    ```
+    
+</details>
```

Comment:
The point is to demonstrate what a code block looks like, there are a few issues here:

- This isn’t going to look as expected if all lines start with four spaces.
- We really need the exact same code block as above, no extra `<br>`, otherwise it really defeats the point of showing what they look like when rendered!

Reply:
@thibaudcolas, thank you for explaining the changes you made. I'll do a better job next time!